frontend/src/features/mapSideBar/tabs/ltsa/LtsaChargeSubForm.tsx

```diff
@@ -18,60 +16,35 @@ export const LtsaChargeSubForm: React.FunctionComponent<ILtsaChargeSubFormProps>
 }) => {
   const { values } = useFormikContext<LtsaOrders>();
   const charges = getIn(values, withNameSpace(nameSpace, 'chargesOnTitle')) ?? [];
-  const [isExpanded, setIsExpanded] = React.useState<boolean>(true);
-
   return (
-    <React.Fragment key={`charge-sub-row-${nameSpace}`}>
-      <StyledSectionHeader>
-        Charges, Liens and Interests
-        {!isExpanded && (
-          <ArrowDropDownIcon
-            onClick={() => {
-              setIsExpanded(!isExpanded);
-            }}
-          />
-        )}
-        {isExpanded && (
-          <ArrowDropUpIcon
-            onClick={() => {
-              setIsExpanded(!isExpanded);
-            }}
-          />
+    <div>
```

Comment:
huh, why are there changes to ltsa charges on this story?

Reply:
I think I see why now, the motivation seems to have been refactoring the FormSections. This is fine but please put a note on your story to the FT team that your story (unexpectedly) affects those screens.

Reply:
The problem was that the collapsable feature was not standardized into a container. These changes allow any Section to be collapsable

Reply:
Yeah, thats fine, I just want us to get into the habit of warning the FT team when we make changes to code that isn't in the scope of the acceptance criteria for the story, since that means they wouldn't run their manual scenarios against those affected pages.